webknossos/pyproject.toml

```diff
@@ -23,6 +23,10 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Visualization",
     "Topic :: Software Development :: Libraries",
     "Typing :: Typed",
+    "Programming Language :: Python :: 3.9", 
```

Comment:
Fixes 
<img width="559" alt="Screenshot 2024-11-22 at 13 38 52" src="https://github.com/user-attachments/assets/1492dae3-024f-42c8-a2e7-cefd40d9fd80">

<img width="548" alt="Screenshot 2024-11-22 at 13 40 59" src="https://github.com/user-attachments/assets/3fe65234-a065-41af-8eb8-0b1b224db308">

Reply:
Alternatively, shields.io can somehow parse the `requires_python` argument from the `pyprpoject.toml` and display a range instead of each version. However, that would not add this info to Pypi.org website where I typicall find it easier to spot at the first glance. 

<img src="https://img.shields.io/badge/dynamic/json?query=info.requires_python&label=python&url=https%3A%2F%2Fpypi.org%2Fpypi%2Fwebknossos%2Fjson">

Code: `https://img.shields.io/badge/dynamic/json?query=info.requires_python&label=python&url=https%3A%2F%2Fpypi.org%2Fpypi%2Fwebknossos%2Fjson`